Add pretrained SfMlearner model

jianganbai opened the pull request:
- SfMlearner is a famous DL solution for depth estimation, first proposed in CVPR 2017

- I have transferred a pytorch pretrained model into a paddle pretrained model and sealed it in a hub module.

- I have run local tests and this hub module functions well.

- Module tutorial can be seen in README.md

Commit:
Add pretrained SfMlearner model

- SfMlearner is a famous DL solution for depth estimation, first propsed in CVPR 2017
- I have transferred a pytorch pretrained model into a paddle pretrained model and sealed it in a hub module.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/PaddlePaddle/PaddleHub?pullRequest=1538) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/PaddlePaddle/PaddleHub?pullRequest=1538) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/PaddlePaddle/PaddleHub?pullRequest=1538) it.</sub>

Commit:
fix parameter name bug

- Previous version cannot be called by commandlines.

nepeplwu commented on modules/thirdparty/image/depth_estimation/SfMlearner/module.py:
```diff
@@ -0,0 +1,89 @@
+import paddle
```
import顺序不符合PEP8规范

系统package > 第三方package > 本地package，相互之间以空行隔开，例如：
```python
import os
import argparse

import paddle
import numpy as np
from paddlehub.module.module import runnable, moduleinfo
from paddle.vision.transforms import Resize
from PIL import Image

from SfMlearner.net import DispNetS
from SfMlearner.utils import tensor2array
```

nepeplwu commented on modules/thirdparty/image/depth_estimation/SfMlearner/module.py:
```diff
@@ -0,0 +1,89 @@
+
+        self.img_height, self.img_width = 128, 416 
+
+    def estimation(self, pretrained, img_dir, output_dir, USE_GPU, output_disp=True, output_depth=False):
```
从用户调用api使用的角度来讲，该接口设计存在以下问题：
1. 不应该由用户指定pretrained进行加载，而是默认情况下就加载预训练好的参数
2. 输入图片不应该是图片的目录，而是图片路径或者是已经读取好的图片格式(np.ndarray)，否则不够灵活
3. USE_GPU建议改为use_gpu

整体设计建议参考：
https://github.com/PaddlePaddle/PaddleHub/blob/release/v2.1/modules/image/keypoint_detection/face_landmark_localization/module.py#L99

nepeplwu commented:
提交了一个无用的文件`.module.swp`

Commit:
Revise SfMlearner API

- Input data can be either np.ndarray or images' addresses.
- Delete redundant .swp file.
